src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt

```diff
@@ -0,0 +1,283 @@
+package fi.metatavu.metaform.server.test.functional.tests
+
+import fi.metatavu.metaform.api.client.models.Metaform
+import fi.metatavu.metaform.api.client.models.MetaformVisibility
+import fi.metatavu.metaform.api.client.models.Script
+import fi.metatavu.metaform.api.client.models.ScriptType
+import fi.metatavu.metaform.server.test.functional.builder.TestBuilder
+import fi.metatavu.metaform.server.test.functional.builder.resources.MetaformKeycloakResource
+import fi.metatavu.metaform.server.test.functional.builder.resources.MysqlResource
+import io.quarkus.test.common.QuarkusTestResource
+import io.quarkus.test.junit.QuarkusTest
+import io.quarkus.test.junit.TestProfile
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+/**
+ * Tests for Scripts functionality
+ */
+@QuarkusTest
+@QuarkusTestResource.List(
+  QuarkusTestResource(MysqlResource::class),
+  QuarkusTestResource(MetaformKeycloakResource::class)
+)
+@TestProfile(GeneralTestProfile::class)
+class ScriptsTestsIT {
```

Comment:
Tests with test builder can also be simplified by using the single-expression syntax.

e.g.

fun testCreateScript() {

-->
fun testCreateScript() = TestBuilder().use { builder ->